lib/mayaUsd/nodes/proxyAccessor.cpp

```diff
@@ -50,6 +51,94 @@
 
 namespace MAYAUSD_NS_DEF {
 
+/*! /brief  Scoped object setting up compute context for accessor
+
+    Proxy accessor supports nested compute that allows injecting DG dependencies to USD. More
```

Comment:
Is there an extra space before "Proxy"?

Reply:
Maybe the other lines have only 3 spaces.

Reply:
Good catch!